.github/workflows/code_quality.yml

```
@@ -0,0 +1,25 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v4
+
+      # Markdown Linter
+      - name: markdownlint-cli2-action
+        uses: DavidAnson/markdownlint-cli2-action@v18.0.0
+        with:
+         config: '.markdownlint.json'
+        globs: '**/*.md'  # prüft nur Markdown Files
```

actionlint (1.7.4)

unexpected key "globs" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)